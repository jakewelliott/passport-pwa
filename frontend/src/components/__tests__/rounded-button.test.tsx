import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import RoundedButton from '../rounded-button';

describe('RoundedButton', () => {
    it('matches snapshot', () => {
        const { container } = render(<RoundedButton title='Test' />);
        expect(container).toMatchSnapshot();
    });
});
