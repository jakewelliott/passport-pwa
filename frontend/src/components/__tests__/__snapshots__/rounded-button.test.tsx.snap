// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`RoundedButton > matches snapshot 1`] = `
<div>
  <button
    class="flex cursor-pointer select-none items-center justify-center text-center rounded-full transition-all duration-200 p-2 w-48 bg-secondary-darkteal"
    type="button"
  >
    <p
      class="text-system-white text-2xl"
    >
      Test
    </p>
  </button>
</div>
`;
